db/migrate/20140505201716_migrate_agents_to_liquid_templating.rb

```diff
@@ -1,31 +1,6 @@
 require 'liquid_migrator'
 
 class MigrateAgentsToLiquidTemplating < ActiveRecord::Migration[4.2]
-  class Agent < ActiveRecord::Base
```

Comment:
I did not find a way to make this work with `zeitwerk`. It complains about the Agent classes having a mismatched superclass, probably because the autoloader already loaded the real Agent class.

Not sure if there is a workaround or of we need one.